extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+import {setStyles} from '../../../src/style';
+import {user} from '../../../src/log';
+import {platform} from '../../../src/platform';
+import {viewerFor} from '../../../src/viewer';
+import {CSS} from '../../../build/amp-app-banner-0.1.css';
+
+const TAG = 'amp-app-banner';
+
+class AmpAppBanner extends AMP.BaseElement {
+
+  /** @override */
+  preconnectCallback(onLayout) {
+    this.preconnect.url('https://itunes.apple.com', onLayout);
```

Comment:
Should we be checking if `ios` or `android` attributes have been set before pre-connecting? Ie. if I only have an iOS app, we shouldn't pre-connect to the play store.